### PR TITLE
#513 mock server requisition data 1

### PR DESCRIPTION
**Pull request by joshxg**
#513 part 1!

So far this is only the requisitions themselves and not any lines. A lot of boiler plate! Lines will simply be copy and pasting pretty much all of this code and replacing CustomerRequisition with CustomerRequisitionLine.

What are the chances we end up using Request and Response rather than supplier and customer? 100% I'd say.

### CODE REVIEW

File: packages/mock-server/src/api/resolvers.ts [hunk 1]
```diff
@@ -95,6 +97,59 @@ const createListResponse = <T>(
 });
 
 export const ResolverService = {
+  requisition: {
+    get: {
+      byId: (id: string): ResolvedRequisition => {
```
Comment (mark-prins) [inline packages/mock-server/src/api/resolvers.ts:102]: I'm not seeing what's wrong - but I'm getting undefined for the `id` if I try to query a specific requisition 🤷

**Review by mark-prins — COMMENTED** [review]
👍  looks good!

**Review by mark-prins — APPROVED** [review]
oops! that last one should have been an approval.

File: packages/mock-server/src/api/resolvers.ts (hunk 1 above)
Comment (joshxg) [reply packages/mock-server/src/api/resolvers.ts:102]: Ah yes I see the issue, sorry - I'll fix that in a later PR

**Pull request merged**